### PR TITLE
NEW FAQ - Dissimilar indication of risk status

### PR DESCRIPTION
**Pull request by dsarkar**

### CODE REVIEW

**Comment by dsarkar** [comment]
image
m

File: src/data/faq.json [hunk 1]
```diff
@@ -623,7 +623,19 @@
...
                 "title": "Risk Assessment",
                 "id": "risk_assessment",
-                "accordion": [
+                "accordion": [
+                    {
+                        "title": "Dissimilar indication of risk status",
+                        "anchor": "dissimilar_indication_of_risk_status",
+                        "active": true,
+                        "textblock": [
+                            "Why do differences in infection risk status appear in different CWA app's, even for close-by smartphones?",
```
Comment (MikeMcC399) [inline src/data/faq.json:632]: app's => apps
(plural does not use an apostrophe)

File: src/data/faq.json [hunk 2]
```diff
@@ -623,7 +623,19 @@
...
+                            "Varying infection risk ratings are caused by several reasons. However, they do not affect the effectiveness of the Corona Warning App (CWA).",
```
Comment (MikeMcC399) [inline src/data/faq.json:633]: Corona Warning App => Corona-Warn-App
(Use the exact name as it appears in the app itself when the mobile device language is set to English.)

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
The German version looks good.
I have made two recommendations for change in the English language version. 🙂

File: src/data/faq.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq.json:632]: ok !

File: src/data/faq.json (hunk 2 above)
Comment (dsarkar) [reply src/data/faq.json:633]: ok !

**Comment by dsarkar** [comment]
@MikeMcC399 excellent, thanks!

**Comment by dsarkar** [comment]
@heinezen review and merge, please

**Review by heinezen — APPROVED** [review]
LGTM

**Pull request merged**